nvflare/fuel/utils/stats_utils.py

```diff
@@ -311,3 +311,23 @@ def new_message_size_pool(name: str, description="", marks=None) -> HistPool:
     if not marks:
         marks = (0.01, 0.1, 1, 10, 50, 100, 200, 500, 800, 1000)
     return HistPool(name=name, description=description, marks=marks, unit="MB")
+
+
+VALID_HIST_MODES = [StatsMode.COUNT, StatsMode.PERCENT, StatsMode.AVERAGE, StatsMode.MAX, StatsMode.MIN]
+
+
+def parse_hist_mode(mode: str) -> str:
```

Comment:
do we have any usages of this method?

Reply:
Another good catch! It was taken out from _parse_mode from net_manager.py, and intended to be used by the StatsViewer tool (to be checked in separate PR). I need to remove _parse_mode from net_manager and use this func.

Reply:
Changed net_manager to use this func.